client/components/social-buttons/facebook.js

```diff
@@ -88,15 +87,15 @@ class FacebookLoginButton extends Component {
 	render() {
 		return (
 			<button className="button" onClick={ this.handleClick }>
-				<span>
-					<SocialLogo className="social-buttons__logo" icon="facebook" size={ 24 } />
```

Comment:
We discussed this in slack. We could use the Facebook logo and add color using CSS but the colored Google logo is a group of shapes which differs from the version on [social-logos](https://github.com/Automattic/social-logos).